server/errors/errors.go

```diff
@@ -26,6 +26,12 @@ var (
 		Description:    "No file/role name is provided to associate an update with.",
 		HTTPStatusCode: http.StatusBadRequest,
 	})
+	ErrBadPagination = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "BAD_PAGINATION",
+		Message:        "The pagination parameters are invalid.",
+		Description:    "Either the start or number parameters could not be parsed as integers greater than or equal to zero.",
```

Comment:
Maybe "The number parameter could not be parsed as an integer greater...", since we don't check the start except to see if it exists?  (If it doesn't, we return `ErrNotFound`, and not `ErrBadPagination`)